concordia/language_model/ollama_models.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2024 DeepMind Technologies Limited.
```

Comment:
Nit: please rename this to `ollama_model.py` to be consistent with the other `xyz_model.py` files.

Reply:
I am wondering about this. the reason for the plural is that ollama is a wrapper for many models, and it allows Llama, mistral, gemma, etc. So, ollama is not a singular model per se. Thoughts?

Reply:
It's a module that contains only one class `OllamaLanguageModel`, if it were plural I'd expect to see many in here? To be honest either way I think being consistent with the other module names is preferable.